accessibility-checker-extension/gulp/gulpFile.js

```diff
@@ -97,7 +97,13 @@ export default class HelpFile extends React.Component<IHelpFileProps> {
         var snippet = '';
 
         if(children.length > 1){
-            children.forEach((element:any) => {snippet = snippet + element.props.children[0] + '\\n'});
+            children.forEach((element:any) => {
+                if(element.props){
+                    snippet = snippet + element.props.children[0] + '\\n'
+                } else {
+                    element + '\\n'
```

Comment:
This line has no side-effect. What is the intent of this?

Reply:
This change is to compose a string for code snippet copy if a element has no children